sass/layout/_sidebars.scss

```diff
@@ -7,6 +7,25 @@
 	padding: $size__default-padding;
 }
 
+.sidebar-nav {
+	margin-right: 0;
+}
+
+.sidebar-nav .current_page_item a {
+	color: #6E9AA2;
```

Comment:
We should always be using color variables, like `$color__link`, otherwise it makes it more difficult to change the colors in a child theme.